static/app/utils/replays/replayDataUtils.tsx

```diff
@@ -70,9 +70,13 @@ export function breadcrumbFactory(
     type: BreadcrumbType.ERROR,
     level: BreadcrumbLevelType.ERROR,
     category: 'exception',
-    message: error['error.value'],
+    message: error['error.value'].join(''),
```

Comment:
Turns out these are typed as `string[]` in the API.
The values would get passed through into react (through `<BreadcrumbItem>` and `breadcrumbs/utils.tsx::getDetails()`) where the values would be squished together without commas.

I fixed the type and then made sure the values work the same way.
If someone had tried to call `message.toUpperCase()` or something it would've exploded.